src/maintainers/CommunityContributionMaintainer.ts

```diff
@@ -18,7 +19,7 @@ export default class CommunityContributionMaintainer extends BaseMaintainer {
 
   onDefine() {
     // define connector
-    this.connector = this.getSingularityConnector();
+    this.connector = this.getCVMFSConnector();
```

Comment:
One way I can think of is to run a separate maintainer which will run alongside CommunityContributionMaintainer.

Reply:
@alexandermichels Just pushed the code with ``SingCVMFSMaintainer``. I know its identical to ``CommunityContributionMaintainer`` but typescript for some reason does not have a clean way of overriding parent class properties. I will compress the code once I figure that out.

Reply:
Couldn't find a way to support multiple connectors within the same maintainer so right now we have 
 - `CommunityContributionMaintainer ` -> `SingularityConnector`
 - `SingCVMFSMaintainer` extends `CommunityContributionMaintainer` -> `SingCVMFSConnector`

Since `SingularityConnector` and `SingCVMFSConnector` are essentially the same barring the syntax changes, I am proposing a`BaseSingularityConnector`. What do you think ?

Since every maintainer is initialized with a connector, I don't see how we can have multiple connectors for a single maintainer but let me know if you have any ideas.

@alexandermichels